src/views/BlogPostsList.vue

```diff
@@ -0,0 +1,119 @@
+<template>
+  <div class="page-container">
+    <div class="header-container">
+      <UserSwitcher :current-user="currentUser" :user-id="currentUser.userId" />
+      <button v-on:click="addNewPost" class="add-new-post-button">New Post</button>
+    </div>
+
+    <div class="content-container">
+      <h1 class="title-text">Blog posts</h1>
+
+      <BlogPostListMenu
+        :active-item="postsType"
+        :menu-items="['All', 'My Posts']"
```

Comment:
Should this ideally be a constant stored somewhere bc the array is recreated everytime the component rerenders?